test/page_object/home_page/home_page.js

```diff
@@ -5,11 +5,14 @@ class HomePage extends BasePage {
   constructor() {
     super();
     this.url = "https://www.apc.com/us/en";
-    this.CountrySelector = new Element("css", "div.sdl-header-apc_lang-selector > a > span:nth-child(2)");
-      }
+    this.CountrySelector = new Element(
+      "css",
+      "div.sdl-header-apc_lang-selector > a > span:nth-child(2)"
+    );
+  }
   open() {
```

Comment:
There is no need to declare `open()` method if your parent class `BasePage` already has it.